doc/build-openbsd.md

```diff
@@ -77,24 +76,23 @@ and preprocessor defines like `waitid()` and `WEXITED` that are not available.)
 To configure with wallet:
 ```bash
 ./configure --with-gui=no --disable-external-signer CC=cc CXX=c++ \
-    BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" \
-    BDB_CFLAGS="-I${BDB_PREFIX}/include" \
```

Comment:
Same as netbsd, don't we have to mention somewhere that, to enable descriptor wallet sqlite3 needs to be installed?